addons/l10n_do/__manifest__.py

```diff
@@ -97,8 +97,7 @@
         'data/account_data.xml',
         'data/account_tax_report_data.xml',
         'data/account.tax.template.xml',
-        # Country States
-        'data/l10n_do_state_data.xml',
```

Comment:
Not sure l10n_do_state_data.xml is removed in the branch

Reply:
It could have been renamed as done in l10n_cr

Reply:
@jva-odoo @sht-odoo @rgo-odoo

Reply:
@jco-odoo We did not remove the file we rename it as you say. plz check here https://github.com/odoo/odoo/pull/32767/commits/7839a84d1fb5805699dd6f7f8ad687efef2fe2bb#diff-dfc36b852636aa3c0137ace250e5b66eR100

Reply:
ok, you are right. forgot to remove 10n_do_state_data.xml from the data folder
@sht-odoo plz rename the file 10n_do_state_data.xml  -> l10n_do_res_partner_title.xml
 using git command.

Reply:
@jco-odoo change done
Thank you